client/httplib/httplib.go

```diff
@@ -660,6 +661,64 @@ func (b *BeegoHTTPRequest) Response() (*http.Response, error) {
 	return b.getResponse()
 }
 
+// ResponseForValue attempts to resolve the response body to value using an existing method.
+// Calls Response inner.
+// If value type is **string or **[]byte, the func directly passes response body into the pointer.
+// Else if response header contain Content-Type, func will call ToJSON\ToXML\ToYAML.
+// Finally it will try to parse body as json\yaml\xml, If all attempts fail, an error will be returned
+func (b *BeegoHTTPRequest) ResponseForValue(value interface{}) error {
```

Comment:
hhhh, may be you could give it a better name :laughing:

Reply:
That's embarrassing😆I thought that was the name of the method